apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/call-to-action-02.php

```diff
@@ -37,7 +37,8 @@
 ';
 
 return array(
-	'__file'  => 'wp_block',
-	'title'   => esc_html__( 'Call to Action', 'full-site-editing' ),
-	'content' => $markup,
+	'__file'     => 'wp_block',
```

Comment:
We might be able to drop these now :thinking:

Reply:
Removed in 4c1b9d353a4617fd42eb77b59c300f00c9d13b8e
`__file` isn't not mentioned in the documentation and isn't used in core when registering patterns. Tested it out and everything worked as expected.